irs-edc-client/src/test/java/org/eclipse/tractusx/irs/edc/client/contract/service/EdcContractAgreementServiceTest.java

```diff
@@ -88,7 +90,7 @@ void shouldReturnContractAgreements() throws ContractAgreementException {
 
         //THEN
         Mockito.verify(restTemplate)
-               .postForEntity(eq("/v2/contractagreements/request"), any(),
+               .postForEntity(eq("https://irs-consumer-controlplane.dev.demo.catena-x.net/data/v2/contractagreements/request"), any(),
```

Comment:
I would recommend to extract this to a variable as the magic string is used twice in the test. Refactoring would be easier

Reply:
I think its better to explicitly write strings in the expected field for tests. This improves readability and avoids bug during refactoring.